jest.config.js

```diff
@@ -120,7 +120,7 @@ module.exports = {
   setupFilesAfterEnv: ['<rootDir>/scripts/test/setupTests.js'],
 
   // A list of paths to snapshot serializer modules Jest should use for snapshot testing
-  // snapshotSerializers: [],
+  snapshotSerializers: ['jest-emotion'],
```

Comment:
I think it's worth challenging whether or not we want to have CSS in our snapshots. On one hand they might not be useful enough without it, but on the other hand do we want to have to update them every time we have a style change now that we also have Chromatic?

Reply:
I personally think that we shouldn't have snapshot testing as a matter of fact, especially since we have Chromatic.

Unit tests should test something meaningful: user interactions, ally tags, some logic that is applied based on props.

I'd rather rewrite these tests to something meaningful when we'll be moving them to a separate component.

Reply:
I like it in places [like this](https://github.com/contentful/forma-36/pull/830/files#diff-64752a5a2d5b11e4e45b39911fff261800a48c187fa1d351e6d0f656901ccd4b) 🙂